package.json

```diff
@@ -22,32 +22,31 @@
         "preversion": "npm run lint",
         "NOTYET-version": "npm run format && git add -A src",
         "NOTYET-postversion": "git push && git push --tags",
-        "prepare": "npm run build"
+        "prepare": "husky install"
     },
     "dependencies": {
-        "@octokit/graphql": "^4.5.6",
-        "@types/luxon": "^1.25.0",
-        "@types/node-fetch": "^2.5.7",
-        "luxon": "^1.25.0",
-        "node-fetch": "^2.6.1",
-        "typescript": "^4.0.3"
+        "@octokit/graphql": "^4.6.1",
+        "@types/luxon": "^1.26.2",
+        "@types/node-fetch": "^2.5.8",
+        "luxon": "^1.26.0",
+        "node-fetch": "^2.6.1"
     },
     "publishConfig": {
         "registry": "https://registry.npmjs.org "
     },
     "devDependencies": {
         "@types/jest": "^26.0.20",
-        "@types/node": "^14.11.2",
-        "@typescript-eslint/eslint-plugin": "^4.3.0",
-        "@typescript-eslint/parser": "^4.3.0",
-        "eslint": "^7.10.0",
-        "eslint-config-prettier": "^6.12.0",
-        "eslint-plugin-prettier": "^3.1.4",
-        "husky": "^4.3.0",
+        "@types/node": "^14.14.35",
+        "@typescript-eslint/eslint-plugin": "^4.18.0",
+        "@typescript-eslint/parser": "^4.18.0",
+        "eslint": "^7.22.0",
+        "eslint-config-prettier": "^8.1.0",
```

Comment:
This is a major release change so make sure it hasn't broken anything.

Reply:
updated eslintrc.json file and addressed error when running `npm lint`